root/usr/libexec/nethserver/spam-training.sh

```diff
@@ -56,10 +57,10 @@ if [ $? -eq 0 ] && ! echo $sa_learn_group | \
     exit 0;
 fi
 
-if ! [ $ACTION == 'ham' ] && ! [ $ACTION == 'spam' ] ; then
+if ! [ $ACTION == 'learn_ham' ] && ! [ $ACTION == 'learn_spam' ] ; then
     log err "Action '${ACTION}' is not recognized" 
     exit 3
 fi
 
-/usr/sbin/sendmail -F 'spam-training.sh script' -r root@`hostname` ${USER}+${ACTION}@spamtrain.nh && log info "Message enqueued as ${ACTION}"
+/usr/bin/rspamc -h localhost:11334 -P $PWD $ACTION  && log info "Message enqueued as ${ACTION}"
```

Comment:
To avoid escaping special chars from the password replace `$PWD` with

     $(< /var/lib/nethserver/secrets/rspamd)